### PR TITLE
Initial CLI poc.

### PR DESCRIPTION
**Pull request by alesj**
Run the new cli jar: java -jar cli.jar ... to get the cool interactive shell

### CODE REVIEW

File: cli/alias.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+alias rscli='java -cp "/Users/alesj/projects/redhat/apicurio-registry/cli/target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar" io.apicurio.registry.cli.EntryCommand'
+java -cp "/Users/alesj/projects/redhat/apicurio-registry/cli/target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar" picocli.AutoComplete -n rscli io.apicurio.registry.cli.EntryCommand
+https://github.com/remkop/picocli/blob/master/picocli-shell-jline2/README.md
+java -jar target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar
+alias rsmain='java -jar /Users/alesj/projects/redhat/apicurio-registry/cli/target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar'
```
Comment (EricWittmann) [inline cli/alias.txt:5]: What is this file?  Since it has `/Users/alesj` paths in it I'm assuming it shouldn't be included in the PR?

**Comment by EricWittmann** [comment]
Can you add a simple README to the new `cli` module with basic usage?

File: cli/alias.txt (hunk 1 above)
Comment (alesj) [reply cli/alias.txt:5]: Yeah, this should go into README -- it's the hints you need to easily run the CLI.

**Comment by alesj** [comment]
README added and that alias.txt removed.

**Pull request merged**